src/controllers/dynakube/activegate/capability/capability_test.go

```diff
@@ -1,587 +1,86 @@
-// 								dynatracev1beta1.StatsdIngestCapability.DisplayName,
-// 								dynatracev1beta1.RoutingCapability.DisplayName,
-// 								dynatracev1beta1.DynatraceApiCapability.DisplayName,
-// 							},
-// 						},
-// 					},
-// 				},
-// 			},
-// 			want: &MultiCapability{
-// 				capabilityBase: capabilityBase{
-// 					enabled:   true,
-// 					shortName: consts.MultiActiveGateName,
-// 					argName: strings.Join([]string{
-// 						dynatracev1beta1.KubeMonCapability.ArgumentName,
-// 						dynatracev1beta1.MetricsIngestCapability.ArgumentName,
-// 						dynatracev1beta1.StatsdIngestCapability.ArgumentName,
-// 						dynatracev1beta1.RoutingCapability.ArgumentName,
-// 						dynatracev1beta1.DynatraceApiCapability.ArgumentName},
-// 						","),
-// 					properties: props,
-// 					Configuration: Configuration{
-// 						SetDnsEntryPoint:       true,
-// 						SetReadinessPort:       true,
-// 						SetCommunicationPort:   true,
-// 						CreateEecRuntimeConfig: true,
-// 						ServiceAccountOwner:    "kubernetes-monitoring",
-// 						ServicePorts: AgServicePorts{
-// 							Webserver: true,
-// 							Statsd:    true,
-// 						},
-// 					},
-// 					initContainersTemplates: []v1.Container{
-// 						{
-// 							Name:            initContainerTemplateName,
-// 							ImagePullPolicy: v1.PullAlways,
-// 							WorkingDir:      k8scrt2jksWorkingDir,
-// 							Command:         []string{"/bin/bash"},
-// 							Args:            []string{"-c", k8scrt2jksPath},
-// 							VolumeMounts: []v1.VolumeMount{
-// 								{
-// 									ReadOnly:  false,
-// 									Name:      trustStoreVolume,
-// 									MountPath: activeGateSslPath,
-// 								},
-// 							},
-// 						},
-// 					},
-// 					containerVolumeMounts: []v1.VolumeMount{{
-// 						ReadOnly:  true,
-// 						Name:      trustStoreVolume,
-// 						MountPath: activeGateCacertsPath,
-// 						SubPath:   k8sCertificateFile,
-// 					}},
-// 					volumes: []v1.Volume{{
-// 						Name: trustStoreVolume,
-// 						VolumeSource: v1.VolumeSource{
-// 							EmptyDir: &v1.EmptyDirVolumeSource{},
-// 						},
-// 					}},
-// 				},
-// 			},
-// 		},
-// 	}
-// 	for _, tt := range tests {
-// 		t.Run(tt.name, func(t *testing.T) {
-// 			if got := NewMultiCapability(tt.args.dynakube); !reflect.DeepEqual(got, tt.want) {
-// 				t.Errorf("NewMultiCapability() = %v, want %v", got, tt.want)
-// 			}
-// 		})
-// 	}
-// }
+import (
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	testNamespace     = "test-namespace"
+	testName          = "test-name"
+	testApiUrl        = "https://demo.dev.dynatracelabs.com/api"
+	expectedShortName = "activegate"
+	expectedArgName   = "MSGrouter,kubernetes_monitoring,metrics_ingest,restInterface,synthetic,beacon_forwarder,beacon_forwarder_synthetic"
+)
+
+func getTestInstance() *dynatracev1beta1.DynaKube {
```

Comment:
could this use `getTestInstanceWithoutCapabilities()`?
maybe change to `buildDynakube(includeCapabilities bool)`